### PR TITLE
fix: usage of Optional.orElseThrow after Map.put

### PR DESCRIPTION
**Pull request by artlomako**
## Change list
- fix broken `KeyEvent`

## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

After the last change in `KeyEvent` class, it's impossible to use the key event, because the following code always throws an exception:
```          
  ofNullable(this.keyCode)
                    .map(x -> map.put("keycode", x))
                    .orElseThrow(() -> new IllegalStateException("The key code must be set"));
```

since `map.put()` returns null for new key.

### CODE REVIEW

File: src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Optional;
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java:22]: there is no need for this import

File: src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java [hunk 2]
```diff
@@ -83,8 +84,8 @@ public KeyEvent withFlag(KeyEventFlag keyEventFlag) {
      */
     public Map<String, Object> build() {
         var map = new HashMap<String, Object>();
-        ofNullable(this.keyCode).map(x -> map.put("keycode", x)).orElseThrow(() -> new IllegalStateException(
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java:86]: @valfirst Could there be other places in the source like this one? I assume the issue may have been added during the recent refactoring

Comment (valfirst) [reply src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java:86]: yes, this is a regression issue, I've found 2 more similar occurrences, I'll open PR soon

Comment (mykola-mokhnach) [reply src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java:86]: thanks. Lets address them all and publish a patch asap

Comment (artlomako) [reply src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java:86]: @mykola-mokhnach I believe I fixed then in my last commit

File: src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java (hunk 1 above)
Comment (artlomako) [reply src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java]: fixed

File: src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java (hunk 2 above)
Comment (valfirst) [reply src/main/java/io/appium/java_client/android/nativekey/KeyEvent.java:86]: I've already addressed `AndroidGeoLocation` issues here: #2146

**Comment by valfirst** [comment]
@artlomako checkstyle is failed

**Pull request merged**